scipy/spatial/distance.py

```diff
@@ -183,9 +183,9 @@ def _convert_to_type(X, out_type):
     return np.ascontiguousarray(X, dtype=out_type)
 
 
-def _filter_deprecated_kwargs(kwargs, args_blacklist):
```

Comment:
Since you're already going through the churn of renaming it, I'd pick a clearer name like `deprecated_names` or perhaps even just `names` since `deprecated` is already in the function name.

Reply:
Sure, but I thought I keep it in line with changes made in pandas and proposed in the links. Besides that, I think `names` is a fine argument name in this case.